server/container/metrics/metrics-es-reporter/src/test/java/org/apache/james/metric/es/ES2ReporterTest.java

```diff
@@ -19,8 +19,17 @@
 
 package org.apache.james.metric.es;
 
-import org.junit.jupiter.api.extension.ExtendWith;
+import org.apache.james.backends.es.DockerElasticSearch;
+import org.apache.james.backends.es.DockerElastichSearchExtension;
+import org.apache.james.util.docker.Images;
+import org.junit.jupiter.api.extension.RegisterExtension;
 
-@ExtendWith(DockerElasticSearch2Extension.class)
 class ES2ReporterTest extends ESReporterContract {
+
+    @RegisterExtension
+    static DockerElastichSearchExtension testExtension = DockerElastichSearchExtension.builder()
+        .dockerES(new DockerElasticSearch(Images.ELASTICSEARCH_2))
```

Comment:
Question : should we think of taking out everything left related to ES2 (could be an other PR), now that we finished to migrate?

Reply:
It does not hurt IMO to show `metrics reporting` works on ES2 and ES6